lib/xiaomiTrvUtils.js

```diff
@@ -0,0 +1,233 @@
+function readTemperature(buffer, offset) {
```

Comment:
Could you move this file into https://github.com/Koenkk/zigbee-herdsman-converters/blob/8be00cc65b20e3eb0f2726d74948f490db6536db/lib/xiaomi.js#L864 ?

Reply:
Moved